Python/pywarpx/Psatd.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2016 David Grote
```

Comment:
You can change the copyright add to your name.

Also, can this file be called PSATD.py? This would be consistent with the method='PSATD'.